### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by carlotrimarchi**

### CODE REVIEW

File: lesson01.md [hunk 1]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+---
+
+### Additional Material
+
+* [javascript.info](https://javascript.info/) - tutorials with nice (often visual) explanations  
+* [freecodecamp.org](https://www.freecodecamp.org/) - full free courses on many topics
+* [udemy.com](https://www.udemy.com): Introduction to JS (course)
+* [codeacademy.com](https://www.codecademy.com): Introduction to JS (another course)
+* [edabit.com](https://edabit.com): Learn JavaScript with interactive challenges and external resources
+
```
Comment (sevtapim) [inline lesson01.md:261]: I recently discovered a new JS course, you have to pay 40 EUR for the complete course, but it is kick-ass compared to freecodecamp and udemy and Codecademy. I wonder shall we add that to the list?
https://learnjavascript.online

File: lesson01.md [hunk 2]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+* `>=` greater or equal
+* `<=` less or equal
+
+All of these return a boolean!
+
+---
+
+### Let's practice
+
+* Are you older than the person next to your right?
```
Comment (sevtapim) [inline lesson01.md]: In Zoom the peson to right is difficult to determine. Maybe are you older than me, or Harald could be used instead?

File: lesson01.md [hunk 3]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+* Do you have the same first name as the person next to you?
+
```
Comment (sevtapim) [inline lesson01.md:527]: Again, maybe do you have the same name as me or Abdullah?

Comment (carlotrimarchi) [reply lesson01.md:527]: yes, you're right about those comments. I wasn't sure as well, but I'll fix it

File: lesson01.md (hunk 1 above)
Comment (carlotrimarchi) [reply lesson01.md:261]: I can add it and specify that it's not for free

File: lesson01.md [hunk 4]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+### Importance of data types
+
+```js
+1 + 1        // is 2
+"1" + "1"    // is "11"
+```
+
+Make sure to always choose the correct data type for your data,
+because the operators in JavaScript behave differently depending on the data type.
```
Comment (haraldF) [inline lesson01.md]:
```suggestion
Make sure to always choose the correct data type for your data,
since the operators in JavaScript behave differently depending on the data type.
```

File: lesson01.md [hunk 5]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+Open the console, and type in the following:
+
+* true
+* false
+
+Yep, that is it.
+
+It seems not very useful...
+
+YOU JUST WAIT!
```
Comment (haraldF) [inline lesson01.md:501]: Would you like to rephrase this as question? E.g. "Why would we dedicate an entire data type for just two values" ?

File: lesson01.md [hunk 6]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+<!-- .element: class="fragment" -->
+
+---
+
+### Summary: Basic Data types 
+
+- numbers
+- strings 
+- booleans 
+
```
Comment (haraldF) [inline lesson01.md]:
```suggestion
- booleans 
- undefined
```

File: lesson01.md [hunk 7]
```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
...
+- what operators have we learnt?
+<!-- .element: class="fragment"  -->
+- what data types have we learnt?
+<!-- .element: class="fragment" -->
+
+---
+
+### Summary: Basic Data types 
+
+- numbers
```
Comment (haraldF) [inline lesson01.md]: do you want to use plural here? `numbers` vs `number` ?

Comment (carlotrimarchi) [reply lesson01.md]: ah, not sure. You think singular is better? And in case, shall I also change it for `strings` and `booleans`?
